client/src/app/app.component.html

```diff
@@ -1,47 +1,19 @@
-<div id="app__main" class="container-fluid h-100">
-  <div id="app__main-states">
-    <div id="states__header" class="h-25">
-      <h1 class="text-center">
-        States
-      </h1>
-    </div>
-    <div id="states__table">
-      <table class="table table-striped table-bordered">
-        <thead class="thead-inverse">
-          <td>_id</td>
-          <td>postalCode</td>
-          <td>Delete</td>
-        </thead>
-        <tbody>
-          <tr *ngFor="let state of states">
-            <td>
-              {{ state._id }}
-            </td>
-            <td>
-              {{ state.postalCode }}
-            </td>
-            <td>
-              <button class="btn" type="button" (click)="onDeleteState(state)">
-            X
-          </button>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-  </div>
-
-  <div id="app__main-submit" class="h-25">
-    <form [formGroup]="stateFormGroup" (ngSubmit)="onSubmitState()">
-      <div class="input-group">
-        <!-- state -->
-        <input type="text" class="form-control" placeholder="postalCode" name="postalCode" formControlName="postalCode" />
 
-        <!-- submit -->
-        <button class=" btn btn-secondary" type="submit">
-      Submit
-    </button>
-      </div>
-    </form>
+<main id="app__main" class="h-100">
+  <header class="header">
+        <nav class="nav nav-pills flex-column flex-sm-row">
```

Comment:
I'm torn between marking or not marking this as work that doesn't belong in this branch because it wasn't mentioned in the design document, only this:

> Add a Static header and footer with a main div in between

And also because this line was mentioned:

> The App Component will bare as possible

However, the first quote just says a static header is present, so it could be interpreted that this is part of the static header that was implied, and the second quote is only defined under the controller section, so the "bare as possible" intent doesn't necessary refer to the template, so can't fault you here.

Not sure whether you want to take action regarding this or not, but in the future we can try to emphasize separation of concerns and managing scope creep or be more clear about intention with a little more description in the design document. What are your thoughts?

Reply:
Let's discuss this in person, I think this is an issue with wording and interpretation.